backends/bmv2/base_test.py

```diff
@@ -33,8 +39,7 @@ class partialmethod(partial):
     def __get__(self, instance, owner):
         if instance is None:
             return self
-        return partial(self.func, instance, *(self.args or ()),
-                       **(self.keywords or {}))
+        return partial(self.func, instance, *(self.args or ()), **(self.keywords or {}))
```

Comment:
I have been using a `.yapf.style` and a `.pylintrc` file with a character limit set to 100. I used 100 because our C code also uses 100 as character limit. I can add this file and its formatting in a separate PR?

I have been not using black since it is a little _too_ opinionated. There are so many options for style tools, I am actually not sure which we should use.

Reply:
I see. Is there any reason to not add the style config files in this PR?

Reply:
This PR is already big, so I was hoping to do the formatting in a separate PR. Then we can also agree on a style there.